.gitignore

```diff
@@ -65,3 +65,6 @@ pip-wheel-metadata
 
 #jupyter notebook
```

Comment:
Could you remove `.ipynb_checkpoints from here. I can see it is already there in other generated files